openstudiocore/src/cli/EmbeddedHelp.hpp

```diff
@@ -7,36 +7,21 @@
 #include <windows.h>
 #endif
 
-extern "C" {
```

Comment:
Yeah it sounds like you found some sort of rb_delayed_provide, this would be better way to do delayed loads, I didn't want to write a custom load wrapper for each extension